lightning_pose_app/ui/project.py

```diff
@@ -720,17 +732,20 @@ def _render_streamlit_fn(state: AppState):
             st.markdown("")
             st.divider()
             st.markdown("")
-            st.markdown("##### Camera views",
-            help= "The support for multiple views is limited to either fusing the views into single frames or utilizing a mirror to generate multiple views from a single camera"
+            st.markdown(
+                "##### Camera views",
+                help="Support for multiple views is currently limited to either fusing the views "
+                "into single frames or utilizing a mirror to generate multiple views from a "
+                "single camera",
             )
-            n_views = st.text_input(
-                "Enter number of camera views:",
-                disabled=not enter_data,
-                value="" if not state.st_project_loaded else str(st_n_views),
+            n_views = st.text_input("Enter number of camera views:",disabled=not enter_data,
+                    value="" if not state.st_project_loaded else str(st_n_views),
```

Comment:
this should revert to the three-line version from before (where each arg is on its own line)